README.md

```diff
@@ -190,6 +190,9 @@ WavefrontSender wavefrontSender = wavefrontClientFactory.getClient();
 ```java
 // Proxy based ingestion
 WavefrontSender wavefrontSender = new WavefrontClient.Builder(proxyServerWithPort).
+        tracingPort(tracing port). // the same port as the customTracingListenerPorts configured in the wavefront proxy
```

Comment:
`customTracingListenerPorts` should be used only when the sender is not used with opentracing sdk or any other app level sdk. Can you please clarify that? same below.

Reply:
Updated. Does that sound better?